content/gophercon-2016/gophercon2016-retrospective.md

```diff
@@ -0,0 +1,50 @@
++++
+author = ["Brian Ketelsen", "Erik St. Martin"]
+date = "2016-07-20"
+title = "GopherCon 2016 - Retrospective"
+tags = ["gophercon","community"]
+series = ["GopherCon 2016"]
+
++++
+
+GopherCon 2016 has come and gone, and in keeping with our tradition we wanted to write up a brief retrospective to capture some of the highlights of the event.  It’s hard to believe it happened so quickly --- we spend months planning all the details and then when the conference arrives it is over in the blink of an eye.  This year we were joined by 1400 Gophers from 38 different countries.  Yes… 38 different countries.  The diversity of people and thoughts at GopherCon was amazing, thanks in part to everyone who donated to the Diversity Scholarship.  We’d like to thank [VividCortex](https://www.vividcortex.com/) especially for their corporate donation.  A community conference should represent the whole community, and our goal is to make the sea of faces at GopherCon look like the faces of Gophers everywhere.  Thanks to everyone who helped get us closer to this goal by purchasing the adorable hoodies.  Together, we helped 37 people come to the conference this year, and that’s no small accomplishment!
```

Comment:
s/come to/attend/ and maybe break it down

"helped 37 people with flights, hotel, or entry tickets where appropriate so they could attend the conference"